charts/csm-authorization/charts/redis/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: redis
+description: A Helm chart for Kubernetes
```

Comment:
Should this say 'A Helm chart for Redis'?

Reply:
Yes, updated.